### PR TITLE
Pull Filers from Institutions Table

### LINKED ISSUE #2654: Update /filers to pull from institutions data
1. Update /filers to pull name, LEI and year from the institutions table, rather than the Transmittal Sheet table.  /filers should be comprised of all institutions who have the HMDA Filer flag set to true in the institutions data. 

2. Double check that all LEIs in the institutions data contain uppercase characters and numbers only (no lowercase letters).

### PR DESCRIPTION
**Pull request by kgudel**
Closes #2654

### CODE REVIEW

File: hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala [hunk 1]
```diff
@@ -12,44 +11,43 @@ import ch.megard.akka.http.cors.scaladsl.CorsDirectives.{
...
               sheets =>
                 sheets
-                  .map(tsEntity =>
-                    HmdaFiler(tsEntity.lei,
-                              tsEntity.institutionName,
-                              tsEntity.year.toString))
+                  .map(instituionEntity =>
+                    HmdaFiler(instituionEntity.lei,
+                              instituionEntity.respondentName,
+                              "2018"))
```
Comment (PatrickHSI) [inline hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala]: can we use the "activityYear" field in lieu of hardcoding 2018 here?

File: hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala [hunk 2]
```diff
@@ -65,13 +63,13 @@ trait ReportingHttpApi extends TsComponent {
...
-          institutionResult <- tsRepository.findByLei(lei)
+          institutionResult <- institutionRepository.findByLei(lei)
         } yield {
           val msaMds =
             msaMdsResult.map(myEntity => MsaMd(myEntity._1, myEntity._2)).toSet
           MsaMdResponse(new HmdaFiler(institutionResult.head.lei,
-                                      institutionResult.head.name,
-                                      institutionResult.head.year + ""),
+                                      institutionResult.head.respondentName,
+                                      "2018"),
```
Comment (PatrickHSI) [inline hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala]: another potential spot for `activityYear`

File: hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala (hunk 1 above)
Comment (kgudel) [reply hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala]: Didn't see that field. Will include.

File: hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala [hunk 3]
```diff
@@ -12,44 +11,43 @@ import ch.megard.akka.http.cors.scaladsl.CorsDirectives.{
...
+            .getAllFilers()
             .map(
               sheets =>
                 sheets
-                  .map(tsEntity =>
-                    HmdaFiler(tsEntity.lei,
-                              tsEntity.institutionName,
-                              tsEntity.year.toString))
+                  .map(instituionEntity =>
+                    HmdaFiler(instituionEntity.lei,
```
Comment (PatrickHSI) [inline hmda-reporting/src/main/scala/hmda/reporting/api/http/ReportingHttpApi.scala]: `instituionEntity.lei.toUpperCase`

**Pull request merged**